### DISCUSSION TITLE
The OAuth 2.0 Client is not allowed to request scope ''openid'

### DISCUSSION THREAD

**Discussion by geozavl** [authelia/authelia#5671] [Q&A] [answered]
I have the following situation: my colleagues requested me to set up Outline (wiki), and since this software mainly relies on external OpenID authentication, I decided to integrate it with Authelia for self-hosted work.

As a result, I ended up with a stack consisting of Traefik, Authelia, Minio, Redis, Postgres, and Outline. On the test server, using docker-compose, this stack successfully took its first breath and started working. Outline was tested and gladly accepted by my colleagues for further deployment on paid cloud resources.

However, when reproducing this stack for the second time on the "service crowded" VM, which caused Traefik HTTPS endpoint usage on a non-standard port (4443), I encountered an Authelia error: **'Authorization Request failed with error: The requested scope is invalid, unknown, or malformed. The OAuth 2.0 Client is not allowed to request scope ''openid''**. The same error applies to other scope combinations such as groups, email, and profile.' Searching for a solution has not led to any constructive results so far, time is passing, and I am already considering deploying an alternative solution or persuading my colleagues to move all this software to a place where we can use a standard HTTPS endpoint.

Both of these solutions come with additional time and material expenses and does not guarantee functionality.

I had already multiple checking the Authelia config, Traefik labels, and Outline variables. Just in case, I'm attaching the configurations, thank you in advance for any constructive feedback.
[out.txt](https://github.com/authelia/authelia/files/12013992/out.txt)
[error.txt](https://github.com/authelia/authelia/files/12013993/error.txt)

**Comment by james-d-elliott** [comment, marked as answer]
It looks to me like Outline is requesting the scope `'openid'` instead of `openid`, i.e. it's adding additional quoting which is invalid.

**Reply by geozavl** [reply]
Thank you so much! 

Configuration string format for Outline changed from:
OIDC_SCOPES="openid profile email"
to
OIDC_SCOPES=openid profile email

**Reply by axelsimon** [reply]
This error appears to come from the Authelia docs (cf. https://www.authelia.com/integration/openid-connect/outline/).

Could it please be corrected there?

**Reply by james-d-elliott** [reply]
It's a community doc, anyone is welcome to update it

**Reply by axelsimon** [reply]
Eventually saw the "edit this page on Github" button and did just that :slightly_smiling_face:
